gating/generate_release_notes/run

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+
+# This script is called by Jenkins to generate release notes.
+
+# The resulting notes must be written to $WORKSPACE/artifacts/release_notes
```

Comment:
This comment is incorrect, and should also be updated in RPCO. The current interface is that release notes should be written to $RE_HOOK_RELEASE_NOTES.